vendor/src/github.com/docker/libnetwork/sandbox/namespace_linux.go

```diff
@@ -4,17 +4,28 @@ import (
 	"fmt"
 	"net"
 	"os"
+	"os/exec"
+	"path/filepath"
 	"runtime"
 	"sync"
 	"syscall"
+	"time"
 
+	log "github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/reexec"
 	"github.com/vishvananda/netlink"
 	"github.com/vishvananda/netns"
 )
 
 const prefix = "/var/run/docker/netns"
```

Comment:
could this be settable? I wonder whether `--exec-root` which defaults to `/var/run/docker` should set this